Extension/src/LanguageServer/settings.ts

```diff
@@ -350,50 +445,58 @@ export class CppSettings extends Settings {
+    public get suggestSnippets(): boolean | undefined { return this.getAsBoolean("suggestSnippets"); }
+    public get intelliSenseEngine(): string | undefined { return this.getAsString("intelliSenseEngine"); }
+    public get intelliSenseEngineFallback(): boolean | undefined { return this.getAsString("intelliSenseEngineFallback") === "enabled"; }
+    public get intelliSenseCachePath(): string | undefined { return this.getAsString("intelliSenseEngine"); }
+    public get intelliSenseCacheSize(): number | undefined { return this.getAsNumber("intelliSenseCachePath"); }
+    public get intelliSenseMemoryLimit(): number | undefined { return this.getAsNumber("intelliSenseMemoryLimit"); }
+    public get intelliSenseUpdateDelay(): number | undefined { return this.getAsNumber("intelliSenseUpdateDelay"); }
+    public get errorSquiggles(): string | undefined { return this.getAsString("errorSquiggles"); }
+    public get inactiveRegionOpacity(): number | undefined { return this.getAsNumber("inactiveRegionOpacity"); }
+    public get inactiveRegionForegroundColor(): string | undefined { return this.getAsString("inactiveRegionForegroundColor"); }
+    public get inactiveRegionBackgroundColor(): string | undefined { return this.getAsString("inactiveRegionBackgroundColor"); }
+    public get autocomplete(): string | undefined { return this.getAsString("autocomplete"); }
+    public get autocompleteAddParentheses(): boolean | undefined { return this.getAsBoolean("autocompleteAddParentheses"); }
+    public get loggingLevel(): string | undefined { return this.getAsString("loggingLevel"); }
+    public get autoAddFileAssociations(): boolean | undefined { return this.getAsBoolean("autoAddFileAssociations"); }
+    public get workspaceParsingPriority(): string | undefined { return this.getAsString("workspaceParsingPriority"); }
+    public get workspaceSymbols(): string | undefined { return this.getAsString("workspaceSymbols"); }
+    public get exclusionPolicy(): string | undefined { return this.getAsString("exclusionPolicy"); }
+    public get refactoringIncludeHeader(): string | undefined { return this.getAsString("refactoring.includeHeader"); }
+    public get simplifyStructuredComments(): boolean | undefined { return this.getAsBoolean("simplifyStructuredComments"); }
+    public get doxygenGeneratedCommentStyle(): string | undefined { return this.getAsString("doxygen.generatedStyle"); }
+    public get doxygenGenerateOnType(): boolean | undefined { return this.getAsBoolean("doxygen.generateOnType"); }
+    public get commentContinuationPatterns(): (string | CommentPattern)[] | undefined {
+        // eslint-disable-next-line no-extra-parens
+        const value = super.Section.get<(string | CommentPattern)[]>("commentContinuationPatterns");
+        if (value === undefined) { return undefined; }
+        if (isArrayOfString(value) || this.isArrayOfCommentContinuationPatterns(value)) {
```

Comment:
You shouldn't need to call `isArrayOfString` here. `isArrayOfCommentContinuationPatterns` should do it.